spec/collectionspace/converter/publicart/acquisition_spec.rb

```diff
@@ -40,69 +40,44 @@
-      context 'when local place' do
+      context 'authority/vocab fields' do
         [
-          "/document/#{p}/owner",
-          "/document/#{p}/acquisitionSource",
+          "/document/#{p}/owners/owner",
+          "/document/#{p}/acquisitionSources/acquisitionSource",
+          "/document/#{p}/acquisitionMethod",
           "/document/#{p}/acquisitionFundingList/acquisitionFunding/acquisitionFundingCurrency",
           "/document/#{p}/acquisitionFundingList/acquisitionFunding/acquisitionFundingSource"
         ].each do |xpath|
           context "#{xpath}" do
-            it 'all values will be URNs' do
-              expect(urn_values(doc, xpath)).not_to include('Not a URN')
+            let(:docurns) { urn_values(doc, xpath) }
+            it 'is not empty' do
+              expect(get_text(doc, xpath)).to_not be_empty
+            end
+
+            it 'values are URNs' do
+              expect(docurns).not_to include('not a urn')
             end
             
             it 'URNs match sample payload' do
-              expect(urn_values(doc, xpath)).to eq(urn_values(record, xpath))
+              expect(docurns).to eq(urn_values(record, xpath))
             end
           end
         end
       end
 
-      context 'text fields in group with local place field' do
+      context 'non-authority/vocab fields in group(s) with overridden field(s)' do
         [
           "/document/#{p}/acquisitionFundingList/acquisitionFunding/acquisitionFundingValue",
           "/document/#{p}/acquisitionFundingList/acquisitionFunding/acquisitionFundingSourceProvisos"
-        ].each do |xpath|
-          context "for xpath: #{xpath}" do
-            it 'is populated' do
-              expect(get_text(doc, xpath)).to_not be_empty
-            end
-            
-            it 'matches sample payload' do
-              expect(get_text(doc, xpath)).to eq(get_text(record, xpath))
-            end
-          end
-        end
-      end
-      
-      context 'when shared place' do
-        let(:attributes) { get_attributes_by_row('publicart', 'acquisition_publicart.csv', 4) }
-        let(:doc) { get_doc(publicartacquisition) }
-        let(:record) { get_fixture('publicart_acquisition_row4.xml') }
-        [
-          "/document/#{p}/owner",
-          "/document/#{p}/acquisitionSource",
-          "/document/#{p}/acquisitionFundingList/acquisitionFunding/acquisitionFundingSource"
         ].each do |xpath|
           context "#{xpath}" do
-            it 'all values will be URNs' do
-              expect(urn_values(doc, xpath)).not_to include('Not a URN')
+            let(:doctext) { get_text(doc, xpath) }
+            it 'is not empty' do
+              expect(doctext).to_not be_empty
             end
             
-            it 'URNs match sample payload' do
-              expect(urn_values(doc, xpath)).to eq(urn_values(record, xpath))
-            end
-          end
-        end
-      end
-      context 'when acquisitionMethod vocab' do
```

Comment:
@Paulmulonzia - Tests for fields populated by authorities and vocabularies can be handled together. No need to test separately.